src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -1,56 +1,182 @@
-import $ from 'jquery';
-import _has from 'lodash/has';
+import _get from 'lodash/get';
+import _map from 'lodash/map';
 import _random from 'lodash/random';
-
+import FixCollection from '../Fix/FixCollection';
+import RouteModel from '../Route/RouteModel';
 import PositionModel from '../../base/PositionModel';
+import {
+    airlineNameAndFleetHelper,
+    randomAirlineSelectionHelper
+} from '../../airline/randomAirlineSelectionHelper';
 import { nm, degreesToRadians } from '../../utilities/unitConverters';
 import { round, sin, cos } from '../../math/core';
-import { distance2d } from '../../math/distance';
-import { bearing, fixRadialDist, inAirspace, dist_to_boundary } from '../../math/flightMath';
-import { vradial, vsub } from '../../math/vector';
+import {
+    bearing,
+    fixRadialDist,
+    isWithinAirspace,
+    calculateDistanceToBoundary,
+    calculateHeadingFromTwoPositions
+} from '../../math/flightMath';
+import { FLIGHT_CATEGORY } from '../../aircraft/AircraftInstanceModel';
+import { AIRPORT_CONSTANTS } from '../../constants/airportConstants';
+import { TIME } from '../../constants/globalConstants';
 import { LOG } from '../../constants/logLevel';
 
 /**
- *  Generate arrivals at random, averaging the specified arrival rate
+ * @property INTERVAL_DELAY_IN_MS
+ * @type {number}
+ * @final
+ */
+const INTERVAL_DELAY_IN_MS = TIME.ONE_HOUR_IN_SECONDS;
+
+// TODO: this shouldn't live here. perhaps move to `FixCollection` as an exported function?
+/**
+ * Encapsulation of a `FixCollection` method.
+ *
+ * This allows for centralization of this logic, while avoiding the need for
+ * another class method.
+ *
+ * @method getFixPostiion
+ * @param fixName {string}
+ * @return fix.position {array}
+ */
+const getFixPosition = (fixName) => {
+    const fix = FixCollection.findFixByName(fixName);
+
+    return fix.position;
+};
+
+/**
+ * Generate arrivals at random, averaging the specified arrival rate
  *
  * @class ArrivalBase
  */
 export default class ArrivalBase {
```

Comment:
In this case, I think this actually makes sense.  The factory figures out which class to instantiate and then that class has only what it needs to create arrivals.  

We could move more of the shared properties to `ArrivalBase`?  There are several that are shared by the inheriting classes that could easily be moved. 

Created an issue for that #55